packages/mint-components/src/components/tax-and-cash/sqm-tax-form/sqm-tax-document-submitted.feature

```diff
@@ -0,0 +1,57 @@
+@owner:andy
+@author:andy
+
+
+Feature: Tax Document Submitted View
+        View that informs the user of their Tax Document submission status.
+
+	Background: A user has submitted their Tax Document
+		Given a user is on the Tax Document Submitted view
+
+	@motivating
+	Scenario: View Tax Document Submitted
+		Given the user has submitted a tax document
+		When they view Document Submitted
+		Then they should see the status of their Tax Document and Banking Information
+
```

Comment:
Does this component display anything while data is loading?

Reply:
What happens if that load fails?

Reply:
I'm not sure if there's a loading state incorporated with designs yet, will have to double check with Shirley